app/controllers/grant_permissions/users_query_controller.rb

```diff
@@ -0,0 +1,14 @@
+module GrantPermissions
+  class UsersQueryController < GrantPermissionsController
+    respond_to :json
+
+    def index
+      @grant = Grant.find(params[:grant_id])
+      @users = User.left_outer_joins(:grant_permissions)
```

Comment:
Why do you need a join to grant_permissions here?

Reply:
This is to remove users from the list that already have an assigned role.

Reply:
don't you exclude them by id? .where.not(id: @grant.grant_permissions.map(&:user_id))

Reply:
I think you might be right now that I look at it. Let me try this without that line. I think you might be right that it would work fine without it.